CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Changed
-Long names in State Spec files
+Long names for carbon diagnostics to use BC, OC, and BR instead of CA.*
```

Comment:
```suggestion
- Long names for carbon diagnostics to use BC, OC, and BR instead of CA.* 
  - Note this requires MAPL v2.35.3+R21C_v1.2.0 or later
```

I hope to make the MAPL tag today (once @elakkraoui gives her blessing) and so we should be nice and specific here.